indico/modules/events/registration/client/js/form/fields/ChoicesSetup.jsx

```diff
@@ -207,38 +202,39 @@ function Choice({
         )}
       </td>
       {!isNoAccommodation && withExtraSlots && (
-        <>
-          <td>
-            <input
-              type="number"
-              name="maxExtraSlots"
-              min="0"
-              value={maxExtraSlots}
-              onChange={makeOnChange('maxExtraSlots')}
-              {...fieldProps}
-            />
-          </td>
-          <td>
-            <input
-              type="checkbox"
-              name="extraSlotsPay"
-              checked={extraSlotsPay}
-              onChange={makeOnChange('extraSlotsPay')}
-              {...fieldProps}
-            />
-          </td>
-        </>
+        <td style={{width: 0}}>
+          <Input
+            type="number"
+            name="maxExtraSlots"
+            min="0"
+            value={maxExtraSlots}
+            onChange={makeOnChange('maxExtraSlots')}
+            icon={{
+              name: 'dollar',
+              link: true,
+              color: extraSlotsPay ? 'blue' : 'grey',
+              title: Translate.string('Extra slots pay'),
+              onClick: () => onChange('extraSlotsPay', !extraSlotsPay),
+            }}
+            style={{width: '7.5em'}}
+            {...fieldProps}
+          />
+        </td>
       )}
-      <td>
-        <input
-          type="checkbox"
-          name="enabled"
-          checked={isEnabled}
-          onChange={makeOnChange('isEnabled')}
-          {...fieldProps}
-        />
-      </td>
-      <td className="row-actions">
+      <td style={{textAlign: 'end'}}>
+        {isEnabled ? (
```

Comment:
Maybe put the toggle after the delete icon - that'd match the action order we have for fields (and in case of no-accommodation the toggle is vertically aligned with the other toggles)